tests/test_isPrimaryFlag.py

```diff
@@ -160,7 +160,7 @@ def setUp(self):
         self.charImResults = charImTask.run(self.exposure)
 
         # set log level so that warnings do not display
-        logging.getLogger("calibrate").setLevel(logging.ERROR)
+        logging.getLogger("lsst.calibrate").setLevel(logging.ERROR)
```

Comment:
The logger name doesn't make it obvious that this is arising from the tests. I guess since it doesn't have `lsst.pipe.tasks` prefix, it may be linked to the unit test?